.travis.yml

```diff
@@ -47,11 +49,12 @@ before_install:
   - etc/testing/travis_cache.sh
 install:
   - etc/testing/travis_install.sh
-script: etc/testing/travis.sh
+script: travis_retry timeout 9m etc/testing/travis.sh; status=$?; if [ $status -ne 0 ]; then etc/testing/kube_debug.sh; fi; exit $status
```

Comment:
A bunch of debug info is already printed, see the `after_failure` section of this file

Reply:
Ah yes! And, as I'm trying to debug test failures in this `travis_retry` block, I'm also interested in seeing debug info before the end of the retries. I shall try to unify these, though.

Reply:
Unified in https://github.com/pachyderm/pachyderm/pull/5137/commits/390ff3589634c8c6fccff17ff27afae92b9a4f3a